bot/resources/fun/latex_template.txt

```diff
@@ -1,4 +1,5 @@
 \documentclass{article}
+\usepackage{amsmath}
```

Comment:
https://github.com/python-discord/sir-lancebot/issues/1057#issuecomment-1131173640
```suggestion
\usepackage{amsmath, amssymb}
```

Reply:
Not sure about this one; I can't remember the last time I or anyone else needed to use amssymb in PyDis.
Although on the other hand, it doesn't really add much compile time, so whatever.

Reply:
Probably best to add it just in case, you never know.

Reply:
+1 for `amssymb`, it has a bunch of useful stuff like `\mathbb` for bold letters and a billion different arrows

Reply:
Alright, I'll add it now.